src/ast/query.rs

```diff
@@ -3340,20 +3340,29 @@ impl fmt::Display for OpenJsonTableColumn {
 /// BigQuery supports ValueTables which have 2 modes:
 /// `SELECT AS STRUCT`
 /// `SELECT AS VALUE`
+///
+/// They can be combined with `[ { ALL | DISTINCT } ]`, e.g.
+/// `SELECT DISTINCT AS STRUCT`
+///
```

Comment:
```suggestion
/// `SELECT [DISTINCT] AS STRUCT`
/// `SELECT [DISTINCT] AS VALUE`
///
```
thinking we can do this instead to flag optionality. Or we can include the specific examples as doc strings on each variant